features/order/managing_orders/browsing_orders/sorting_orders_by_number.feature

```diff
@@ -20,13 +20,13 @@ Feature: Sorting orders by their number
         And the customer chose "Free" shipping method to "United States" with "Offline" payment
         And I am logged in as an administrator
 
-    @ui
+    @ui @api
     Scenario: Orders are sorted by descending numbers by default
         When I browse orders
         Then I should see an order with "#000000001" number
         But the first order should have number "#000000003"
 
-    @ui
+    @ui @api
     Scenario: Changing the number sorting order
         Given I am browsing orders
         When I switch the way orders are sorted by number
```

Comment:
Those scenarios should be tagged with `@no-api` since there is no default sorting.
Instead, you should make two new `@api @no-ui` scenarios and explicitly sort one way and the other.